CHANGELOG

```diff
@@ -1,3 +1,8 @@
+qtile X.X.X, released XXXX-XX-XX:
+    * features
+        - added `get_default_terminal` in utils
+        - added keybinding cheet sheet image generator
```

Comment:
It's no big deal but this line should rather be in the other PR :)

Reply:
I've already squashed all changes in that PR and it is waiting for merge. Sorry but I wanted to save somehow ;)
Next time I'll be more accurate

Reply:
Thanks for adding this; we should probably go back and look at the other stuff we've added, since people asked me about release notes for the last release :)